Source/Particles/RigidInjectedParticleContainer.cpp

```diff
@@ -386,16 +386,12 @@ RigidInjectedParticleContainer::PushP (int lev, Real dt,
 
     if (do_not_push) return;
 
-    const std::array<Real,3>& dx = WarpX::CellSize(lev);
-
```

Comment:
Fixes the warning:
`./Source/Particles/RigidInjectedParticleContainer.cpp:389:31`: warning: unused variable ‘dx’ [-Wunused-variable]
     const std::array<Real,3>& dx = WarpX::CellSize(lev);